### PR TITLE
update install guide; add getting help; pkg name lowercase

### PR DESCRIPTION
**Pull request by pawelru**
- update install guide - added back install from GH as this is still a standard way of installing non-CRAN packages. I added install via `pak` and not `remotes` because this is also standard for pkgs from tidyverse or r-lib.
- change urls to teal gallery and tlg-catalog
- added Getting Help section with reference to slack channel
- replace `browseVignettes(package = "teal")` with url as I believe that most of users will read README from within pkgdown webpage.
- replace "Teal" with "teal" to keep consistent package name
- UPDATE: added "Usage section" with gif recording and move all "Please refer to ... for more" under that section

Please add your feedback. I would like to treat this PR as a reference to the issue for other package updates.

### CODE REVIEW

**Review by m7pr — APPROVED** [review]

File: vignettes/teal-bs-themes.Rmd [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+  %\VignetteIndexEntry{teal and Bootstrap Themes}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
 ---
 
 ## Introduction
 
-We offer the easy application of a custom bootstrap theme to a `teal::init` app. 
-Teal is a consumer of the `bslib` R package which provides tools for customizing Bootstrap themes e.g. of Shiny apps.
+We offer the easy application of a custom bootstrap theme to a `teal::init` app.
```
Comment (chlebowa) [inline vignettes/teal-bs-themes.Rmd:14]:
```suggestion
We offer easy application of a custom bootstrap theme to a `teal::init` app.
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Teal: Interactive Exploratory Data Analysis with Shiny Web-Applications <a href='https://insightsengineering.github.io/teal/'><img src="man/figures/teal.png" align="right" height="139" style="max-width: 100%;"/></a  >
+# teal: Interactive Exploratory Data Analysis with Shiny Web-Applications <a href='https://insightsengineering.github.io/teal/'><img src="man/figures/teal.png" align="right" height="139" style="max-width: 100%;"/></a  >
```
Comment (gogonzo) [inline README.md:1]: What do you think about removing image and keeping duck as icon :D

```suggestion
# teal 🦆: Interactive Exploratory Data Analysis with Shiny Web-Applications  
```

Comment (pawelru) [reply README.md:1]: I am good with emoji icon but would like to keep the hexagon badge also

File: README.md [hunk 3]
```diff
@@ -50,21 +50,31 @@ A lot of the functionality of the `teal` framework derives from the following pa
...
 
-From July 2023 `insightsengineering` packages are available on [r-universe](https://r-universe.dev/).
+```r
+install.packages("teal", repos = c("https://insightsengineering.r-universe.dev", getOption("repos")))
+
+# install.packages("pak")
+pak::pak("insightsengineering/teal@*release")
+```
+
+Alternatively you might also use the development version.
```
Comment (kartikeyakirar) [inline README.md:66]:
```suggestion
Alternatively, you might also use the development version.
```

File: vignettes/teal-bs-themes.Rmd [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
+  %\VignetteIndexEntry{teal and Bootstrap Themes}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
 ---
 
 ## Introduction
 
-We offer the easy application of a custom bootstrap theme to a `teal::init` app. 
-Teal is a consumer of the `bslib` R package which provides tools for customizing Bootstrap themes e.g. of Shiny apps.
+We offer easy application of a custom bootstrap theme to a `teal::init` app.
```
Comment (kartikeyakirar) [inline vignettes/teal-bs-themes.Rmd:14]:
```suggestion
We offer an easy application of a custom bootstrap theme to a `teal::init` app.
```

**Comment by pawelru** [comment]
UPDATE: Added Usage section with gif recording

**Review by kartikeyakirar — APPROVED** [review]
LGTM 👍

**Review by averissimo — APPROVED** [review]
LGTM :+1

**Comment by pawelru** [comment]
I made a mistake in this PR. Please have a look also at the follow-up PR here: https://github.com/insightsengineering/teal/pull/894

**Pull request merged**